integration-tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@slack/bolt-integration-tests",
+  "private": true,
+  "description": "Integration tests for Bolt",
+  "author": "Slack Technologies, Inc.",
+  "license": "MIT",
+  "scripts": {
+    "test": "npm run test:typechecker",
+    "test:typechecker": "dtslint types"
+  },
+  "dependencies": {
+    "@slack/bolt": "^1.4.1",
+    "@types/node": "^8.10.43"
```

Comment:
we could probably update this to the first v10 LTS release, since that's the oldest node version Bolt currently supports.